### PR TITLE
 feat: update styling of event handler property

### PR DESCRIPTION
**Pull request by tilmx**
Adds a new styling to the event handler property.
Those properties now have a box styling to make it clear that the content is connected:

bildschirmfoto 2018-06-13 um 22 18 26

### CODE REVIEW

File: src/components/property-box/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Color } from '../colors';
+import { Copy } from '../copy';
+import { Headline } from '../headline';
+import * as React from 'react';
+import { getSpace, SpaceSize } from '../space';
+import styled from 'styled-components';
+
+export interface PropertyBoxProps {
+	headline?: string;
```
Comment (marionebl) [inline src/components/property-box/index.tsx]: Let's make `headline` and `copy` required

Comment (tilmx) [reply src/components/property-box/index.tsx]: Fixed that!

File: src/components/property-box/pattern.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+	"name": "box-item",
```
Comment (marionebl) [inline src/components/property-box/pattern.json]: Should be `property-box`

Comment (tilmx) [reply src/components/property-box/pattern.json]: Done!

**Pull request merged**